src/test/java/com/redhat/rhjmc/containerjfr/net/web/handlers/CorsEnablingHandlerTest.java

```diff
@@ -93,6 +98,16 @@ void availabilityShouldDependOnEnvVar(boolean available) {
         MatcherAssert.assertThat(handler.isAvailable(), Matchers.equalTo(available));
     }
 
+    @Test
+    void shouldSetCustomOrigin() {
+        Mockito.when(
+                        env.getEnv(
+                                CorsEnablingHandler.ENABLE_CORS_ENV,
+                                CorsEnablingHandler.DEV_ORIGIN))
+                .thenReturn("http://localhost:9065");
+        MatcherAssert.assertThat(handler.getOrigin(), Matchers.equalTo("http://localhost:9065"));
```

Comment:
This just tests that the utility method `getOrigin` returns the expected value, but not that the CORS origin header is actually set. If you look at the `HandlingTest`s lower down in this same file, ex. `shouldAddHeadersToCORS`, you could copy how that test works and assert that the CORS origin header is actually added to the HTTP response with the expected origin.